dotnet-authserver/src/TeacherIdentity.AuthServer/DataAnnotationsExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using System.ComponentModel.DataAnnotations;
```

Comment:
Mostly taken from https://stackoverflow.com/questions/52321148/conditional-validation-in-mvc-net-core-requiredif/59175740#59175740